commands/version/__tests__/version-command.test.js

```diff
@@ -134,6 +151,17 @@ describe("VersionCommand", () => {
       expect.assertions(1);
     });
 
+    it("does not throw if current ref is already tagged when using --force-publish", async () => {
+      checkWorkingTree.mockImplementationOnce(() => {
+        throw new Error("released");
+      });
+
+      const testDir = await initFixture("normal");
+      await lernaVersion(testDir)("--force-publish");
+
+      expect.assertions(0);
```

Comment:
I would prefer this take make an active assertion, such as `expect().not.toThrow()`. Also, unclear why this test is in this PR?

Reply:
Regarding why this test is here, it's because this fixed [happened to be fixed](https://github.com/lerna/lerna/pull/1991/files/255aeb18db2035d663079be0f9daf9cc02b7df49#r271093980) here. Will address assertion style.

Reply:
Making sure this particular call *doesn't* throw is just a bad approach in general. Going to test for a positive outcome instead.

Reply:
Fix: https://github.com/lerna/lerna/pull/1991/commits/a9ba5dfc2208f5d88a4d724a9394a84cba29435a